riocli/usergroup/inspect.py

```diff
@@ -43,13 +44,38 @@ def inspect_usergroup(ctx: click.Context, format_type: str, group_name: str, gro
         org_guid = ctx.obj.data.get('organization_id')
         usergroup = client.get_usergroup(org_guid, group_guid)
         data = make_usergroup_inspectable(usergroup)
-        inspect_with_format(data, format_type)
+        inspect_with_format(transform_usergroup(data, org_guid), format_type)
     except Exception as e:
         click.secho(str(e), fg=Colors.RED)
         raise SystemExit(1)
 
 
-def make_usergroup_inspectable(usergroup: typing.Any):
+def transform_usergroup(usergroup: typing.Dict, org_guid: str) -> typing.Dict:
```

Comment:
Do we need two functions? I think we can merge make_usergroup_inspectable and transform_usergroup. Also, please remove make_project_inspectable if not used anywhere.

Reply:
Removed all the helpers. Renamed the transform helper as `to_manifest`.